.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "External/mINI"]
 	path = External/mINI
 	url = https://github.com/pulzed/mINI.git
+[submodule "External/discord-rpc"]
+	path = External/discord-rpc
+	url = https://github.com/discord/discord-rpc.git
```

Comment:
It looks like this is deprecated and you should instead be using the Discord GameSDK instead. You can probably incorporate that using `ExternalProject_Add` to download the SDK files and configure the project.